android/app/build.gradle

```diff
@@ -6,6 +6,7 @@ apply plugin: 'realm-android'
 
 project.ext.envConfigFiles = [
     debug: ".env",
+    staging: ".env.beta",
```

Comment:
should this be `.env.staging` now since you added that file?

Reply:
😬 yup